en/overview/javaworldwind_overview.rst

```diff
@@ -13,23 +13,14 @@ Java World Wind
 Desktop Virtual Globe
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. Cameron Comment: This implies that Java World Wind is not a product in its
-  own right, but just an API. Ie, before I can use Java World Wind, I need to
-  write a program. Is that correct? (This would be an issue for many new users)
 
-Java World Wind provides an Application Programming Interface (API) to display and interact with a virtual globe. 
+Java World Wind is a virtual globe containing a rich set of features for displaying and interacting with geographic data and representing a wide range of geometric objects thanks to  an Application Programming Interface (API).

```

Comment:
Changes I'm about to apply (feel free to suggest alternative wording). I'm removing reference to the API in the introduction sentence. It is covered prominently below.